ApplicationCode/ProjectDataModel/RimGridView.cpp

```diff
@@ -148,6 +148,8 @@ RimGridView::~RimGridView( void )
 
     delete this->m_overlayInfoConfig();
 
+    delete m_surfaceResultDefCollection;
```

Comment:
The delete is now unnecesary and should really be removed for all the ChildFields and ChildArrayFields in this destructor.